src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,130 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .m-dropdown {
+    position: relative;
+    display: inline-block;
+
+    .m-dropdown__menu {
+      position:absolute;
+      left: 0;
+      min-width: 100%;
+    }
+  }
+
+  .m-dropdown__trigger {
```

Comment:
@dani-cl-madrid This should work inside .m-dropdown with &__trigger

Reply:
If I do that, it will make example n. 2 of documentation not to work. Should I remove that example?

Reply:
@jllr This is by design. We need to support example 2 which allows for the trigger to render outside the scope of m-dropdown

Reply:
I was wrong. It will work the way Jose says.